sm/download/Cargo.toml

```diff
@@ -9,6 +9,7 @@ description = "download_manager"
 [dependencies]
 backoff = { version = "0.3", features = ["tokio"] }
 json_sm = { path = "../json_sm" }
+libc = "0.2.107"
```

Comment:
I'm surprised to see that dependency just for a single type def.

* Why is the proper type not managed by the `nix` crate?

Reply:
Fixed, replaced with nix::libc::off_t